index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" href="http://fonts.googleapis.com/css?family=Roboto">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <header class="navbar">
+    <a class="logo" href="#">Marco</a>
+    <img class="hamburger" src="images/hamburger.png" alt="">
+  </header>
+
+  <section class="headline">
+    <h1>Hey There. <br> I'm Marco</h1>
+    <h2>I am a Software Developer</h2>
+    <p>I can help build you a product, feature or website. Look through some of my work and experience! If you like what
+      you see and have a project you need coded, don't hesitate to contact me.</p>
+    <img class="background" src="images/background.png" alt="">
+  </section>
+
+  <section class="social-icons">
+    <img src="images/github.png" alt="">
+    <img src="images/linkedin.png" alt="">
+    <img src="images/behance.png" alt="">
+    <img src="images/twitter.png" alt="">
+    <img src="images/medium.png" alt="">
+  </section>
```

Comment:
- [ ] To also enforce the good use of the semantic tag, kindly change this to `ul` and wrap each image in an `li` tag and also, wrap each `img` in the `a` tag to make them clickable as this should be links to your social media platform.

`<ul ><li><a href="#"><img ></a></li></ul>`

- [ ] After making the changes, kindly move the ul and its contents inside the `<section class="headline"</section>` as these icons are meant to be in this section.